ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/objectstore/TestJBTM3262.java

```diff
@@ -39,15 +43,25 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.fail;
+import static org.junit.Assume.assumeTrue;
 
 /**
  * Verify that a log created by the code prior to the fix for JBTM-3249 can be read by later code
  */
-public class TestJBTM3249 {
+public class TestJBTM3262 {
```

Comment:
Is it really good to name the test based on the Jira name? It's more common to name the testcase base on what functionality it provides.

Reply:
A more accurate name would be

```java
read_records_created_before_sha_0b511d470f7ab8dedf585cb14d4900170684621b
```

Anything else is not going to capture the intent of the test - namely to verify that JBTM3262 does not break backwards compatibility.

Other than this is this "PR good to go".